dev/domain-controller/new-agent-vm/Install-PCoIPAgent.ps1

```diff
@@ -372,6 +375,74 @@ Configuration InstallPCoIPAgent
 			}
 		}
 
+		Script StartCAMIdleShutdownService
+        {
+            DependsOn  = @("[Script]StartPcoIPService")
+
+            GetScript  = { return 'Start CAM Idle Shutdown Service'}
+
+            TestScript = { 
+				$serviceName = "CAMIdleShutdown"
+				
+				if ([string]::IsNullOrWhiteSpace($using:enableAutoShutdown)) {
+					return $true
+				}
+
+				try {
+					$svc = Get-WmiObject -Class Win32_Service -Filter "Name='$ServiceName'"
```

Comment:
You're right - need to add the error action for the case where the service isn't installed